packages/spor-alert-react-native/src/BaseAlert.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  composeRestyleFunctions,
+  createVariant,
+  spacing,
+  SpacingProps,
+  spacingShorthand,
+  SpacingShorthandProps,
+  useRestyle,
+  VariantProps,
+} from "@shopify/restyle";
+import React, { useEffect, useState } from "react";
+import { Box } from "@vygruppen/spor-layout-react-native";
+import { Theme } from "@vygruppen/spor-theme-react-native";
+import { Text } from "@vygruppen/spor-typography-react-native";
+
+type Variant = VariantProps<Theme, "alertColorScheme", "colorScheme">;
+const colorScheme = createVariant({
+  themeKey: "alertColorScheme",
+  property: "colorScheme",
+});
+
+type RestyleProps = SpacingProps<Theme> &
+  SpacingShorthandProps<Theme> &
+  Variant;
+
+const restyleFunctions = composeRestyleFunctions<Theme, RestyleProps>([
+  spacing,
+  spacingShorthand,
+  colorScheme,
+]);
+
+export type ColorVariants =
+  | "yellow"
+  | "light-yellow"
+  | "orange"
+  | "red"
+  | "green"
+  | "blue";
+
+type AlertProps = Exclude<RestyleProps, "variant"> & {
+  children?: React.ReactNode;
+  colorScheme: ColorVariants;
+  icon: JSX.Element;
+  text: React.ReactNode;
```

Comment:
Denne synes jeg kan hete `title`, som resten (evnt rename alt til `heading`. Og om man ønsker den bold, så kan man spesifisere det selv slik:

```tsx
<BaseAlert heading={<Text fontWeight="bold">Overskrift</Text>}>...</BaseAlert>
```

Det er litt mer klønete enn å ha en egen `weight` prop, men siden det kun er våre egne interne komponenter som skal spesifisere det, så er ikke det noe stort problem. Og om man ønsker mer fleksibilitet i fremtiden (underlines? kursiv? annen farge?) så slipper man å legge til egne props for det.

Reply:
Enig, Heading blir implementert 😄